projects/packages/codesniffer/changelog/update-php-requirements

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Again, major version bump.

Reply:
👍 I switched to a major version bump in 6caa55c72efd1f4cd8352fb39b30055590d6efbc